test-data/unit/check-modules.test

```diff
@@ -670,45 +670,45 @@ def f(x: str) -> None: pass
 [case testTypeCheckNamedModule]
 # cmd: mypy -m m.a
 [file m/__init__.py]
-None + 1
```

Comment:
> Are you sure it was done to the pytest tests?

I am not, but I assumed this, since the tests were passing before, i.e. were giving an error for ``None + 1``.